comp/dogstatsd/server/server_test.go

```diff
@@ -214,814 +99,6 @@ func TestNoRaceOriginTagMaps(t *testing.T) {
 	}
 }
 
-func testReceive(t *testing.T, conn net.Conn, demux demultiplexer.FakeSamplerMock) {
```

Comment:
Broken up into TestMetricTypes, TestMetricPermutations, TestEvents, and TestServiceChecks in server_worker_test.go